pegasus/forms/hoc_hardware_prizes_2015.rb

```diff
@@ -0,0 +1,26 @@
+require pegasus_dir 'forms/hoc_hardware_prizes_2014'
+
+class HocHardwarePrizes2015 < HocHardwarePrizes2014
+  def self.normalize(data)
+    result = {}
+    result[:email_s] = required email_address data[:email_s]
+    result[:name_s] = required stripped data[:name_s]
+    result[:school_name_s] = required stripped data[:school_name_s]
+    result[:school_address_s] = required stripped data[:school_address_s]
+    result[:school_type_s] = required enum(data[:school_type_s].to_s.strip.downcase, ['elementary', 'middle_school', 'high_school'])
+    result[:qualifying_school_b] = required stripped data[:qualifying_school_b]
+    result[:title_I_school_b] = nil_if_empty stripped data[:title_I_school_b]
```

Comment:
I added two new fields, so I think I have to copy all fields over to 2015? Or do I list all of the existing 2014 fields in 2014 and the two new fields in this one?